config/jobs/kubernetes/sig-release/release-branch-jobs/1.11.yaml

```
@@ -449,6 +459,9 @@ periodics:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.11-all, sig-release-1.11-blocking, sig-node-kubelet
```

if order of dashboards matters...
```suggestion
      testgrid-dashboards: sig-release-1.11-blocking, sig-release-1.11-all, sig-node-kubelet
```